### PR TITLE
Fix the AsciiDoc files in the docs folder that are causing build errors and warnings

### PR DESCRIPTION
**Pull request by sgilda**
When you run 'mvn clean install' from the 'docs/' folder, the build is now clean with no errors or warnings.

You might want to review this with the '-w' switch to ignore white space changes.

Let me know if I need to create a JIRA for this one.

### CODE REVIEW

**Comment by bstansberry** [comment]
This is ok to test.

File: docs/src/main/asciidoc/_developer-guide/How_do_I_migrate_my_application_from_AS7_to_WildFly.adoc [hunk 1]
```diff
@@ -621,25 +621,24 @@ actual ha-policy of the migrated server. In that case, we discard it and
 the user will have to add the correct ha-policy afterwards (ha-policy is
 a single resource underneath the messaging-activemq's server resource).
 
-* Can not migrate attribute local-bind-address from resource X. Use
-instead the socket-binding attribute to configure this
-discovery-group.Can not migrate attribute local-bind-port from resource
-X. Use instead the socket-binding attribute to configure this
+* Can not migrate attribute local-bind-address from resource X. Use instead the socket-binding attribute to configure this discovery-group.Can not migrate attribute local-bind-port from resource X. Use instead the socket-binding attribute to configure this
```
Comment (bstansberry) [inline docs/src/main/asciidoc/_developer-guide/How_do_I_migrate_my_application_from_AS7_to_WildFly.adoc]: This looks wrong. This PR didn't make it wrong though. :)

**Review by bstansberry — APPROVED** [review]

**Comment by sgilda** [comment]
@bstansberry : I had only planned to fix the documentation build errors, but the conversion from Confluence to AsciiDoc resulted in awkward carriage returns and some formatting issues. The diff for the `How_do_I_migrate_my_application_from_AS7_to_WildFly.adoc` file looks bigger than it really is due to the removal of carriage returns. This topic still needs more editing for language, but I think it formats correctly now. This is ready for review again.

**Review by bstansberry — APPROVED** [review]

**Comment by bstansberry** [comment]
retest this please

**Comment by bstansberry** [comment]
Thanks!

**Comment by sgilda** [comment]
@bstansberry : Thanks for the merge !

**Pull request merged**